src/Serialization/containers.jl

```diff
@@ -132,7 +173,9 @@ function load_object(s::DeserializerState, T::Type{<:Matrix{S}}) where S
   end
 end
 
-function load_object(s::DeserializerState, T::Type{<:Matrix{S}}, params::NCRing) where S
+load_object(s::DeserializerState, T::Type{<:Matrix{S}}, ::Nothing) where S = load_object(s, T)
+
+function load_object(s::DeserializerState, T::Type{<:Matrix{S}}, params::U) where {S, U}
```

Comment:
```suggestion
function load_object(s::DeserializerState, T::Type{<:Matrix{S}}, params) where {S}
```
since you don't use `U` anywhere, you don't have to specialize to it